src/aggregation/agg_req.rs

```diff
@@ -36,7 +37,8 @@
 //!       "ranges": [
 //!         { "from": 3.0, "to": 7.0 },
 //!         { "from": 7.0, "to": 20.0 }
-//!       ]
+//!       ],
+//!       "keyed": false
```

Comment:
this is the default, I think we can just omit it

Reply:
It didn't pass the tests...
https://github.com/quickwit-oss/tantivy/runs/7517376126?check_suite_focus=true

Maybe because I removed the below line?
https://github.com/quickwit-oss/tantivy/pull/1424/commits/5ab5f070edb5ae7e2f00daeb626e517e8290d83a#

Reply:
Ah I okey, it seems I just need to use `#[serde(default)]`🙏